assets/sass/patterns/molecules/personalised-cover-download.scss

```diff
@@ -1,20 +1,132 @@
 @import "../../definitions";
 
-.personalised-cover-download-wrapper {
-  @include flexbox-centre-x-and-y();
+.wrapper-alternative {
+  @include wrapper-alternative  ();
+}
+
+.site-header__title-border {
+  @include margin(48, "top");
+  float: left;
+  position: relative;
+  z-index: 21;
+}
+
+.personalised-cover-outer {
+  @include margin(72, "bottom");
 }
 
 .personalised-cover-download {
-  @include wrapper();
-  text-align: center;
   @include blg-pad-vertical-large-to-extra-large();
+  max-width: 735px;
 
   > :last-child {
     margin-bottom: 0;
   }
 }
 
+.personalised-cover__header-text {
+  @include margin(70 0 72 0);
+  text-align: center;
+}
+
+.personalised-cover-text {
+  @include padding(24, "bottom");
+}
+
+.personalised-cover-thumbnail {
+  margin: 0 auto 48px;
+  max-width: 348px;
+
+  img {
+    max-width: 100%;
+  }
+}
+
+.personalised-cover-buttons {
+  display: grid;
+  justify-content: center;
+}
+
+.form-item__checkbox {
+  margin: 0 auto 48px;
+  grid-row: 1;
+}
+
+.button-collection {
+  display: grid;
+  grid-row: 2;
+  justify-items: center;
+}
+
+.button-collection__item {
+  @include margin(12, "left");
+  @include margin(12, "right");
+}
+
 .personalised-cover-download__icon {
-  max-width: 100%;
-  @include blg-spacing("bottom", "small");
+  filter: drop-shadow(0 0 0.75rem rgba(0,0,0,0.2));
+}
+
+.button--variation {
+  @include colors($color-primary);
+  @include padding(18 18 14);
+  font-weight: 500;
+  letter-spacing: 0.4px;
+
+  &:hover {
+    @include colors($color-primary-dark);
+  }
+}
+
+@media only screen and (min-width: #{get-rem-from-px($bkpt-site--small)}em) {
+
+  .button-collection {
+    display: flex;
+  }
+
+}
+
+@media only screen and (min-width: #{get-rem-from-px($bkpt-site--medium)}em) {
+
+  .personalised-cover-outer {
+    display: grid;
+    grid-template-columns: 50%;
+    margin: 0 auto 72px;
+    max-width: 780px;
+  }
+
+  .personalised-cover__header-text {
+    @include margin(94, "top");
+  }
+
+  .personalised-cover-text {
+    @include padding(20, "right");
+    grid-column: 1;
+    grid-row: 1;
+  }
+
+  .personalised-cover-thumbnail {
+    @include margin(24, "left");
+    @include margin(24, "bottom");
+    grid-column: 2;
+    grid-row: 1/3;
+    justify-self: end;
+    max-width: 324px;
+  }
+
+  .personalised-cover-buttons {
+    @include margin(-72, "right");
+    justify-content: left;
+
```

Comment:
nitpick: empty line. You can ignore this. Maybe we should capture a requirement for a linter.